lib/index.js

```diff
@@ -24,24 +24,54 @@ class Client {
 
     // Get the JWT token from CkanAuthz, the user should provide the url
     if (url) {
-      const response = await CkanAuthApi.getJWTFromCkanAuthz(url, this.authToken, scope)
+      const response = await CkanAuthApi.getJWTFromCkanAuthz(
```

Comment:
this is code prettification. It would be easier if this were a separate PR 😄

Reply:
Yea totally agree, my bad sorry